tests/requirements.txt

```diff
@@ -3,6 +3,7 @@ django-filer>=1.4.0
 djangocms-picture>=2.1.0
 djangocms-link>=2.2.1
 django-polymorphic>=2.0.3
+django-treebeard<4.5
```

Comment:
well yes, for [testing legacy releases](https://travis-ci.org/github/django-cms/djangocms-text-ckeditor/jobs/765064626).

Reply:
It's a good point. Could we add a comment about it to the line?

For me one of the hardest parts of upgrading a python project is figuring out which of the 100 requirements were frozen accentually, for an outdated reason, or because unfreezing them is going to break the page number 583 :)

Reply:
A side note on that... If you've not come across pip-tools then it makes upgrading packages really simple by managing all the compatibility for you.

Reply:
Sure, I've been using pip-tools and poetry for years now. The issue is a bit subtle though: a developer can see that somebody 3 years ago added a package that isn't functional anymore and breaks the server at the moment. Is that package needed now? Why was it added? Does he need to spend his workday forking its git or testing hundreds of pages after its deletion?